server/api/organisation/organisation.controller.js

```diff
@@ -45,8 +45,13 @@ const putOrganisation = async (req, res) => {
 
   await Organisation.findByIdAndUpdate(req.params._id, { $set: req.body })
   res.json(await Organisation.findById(req.params._id).exec())
+  // Domain string validation
+  if (Object.keys(Organisation).includes('domainName')) {
```

Comment:
Is `Object.keys(Organisation)` meant to be `Object.keys(req.body)`?

Reply:
Made the change to res.body